siteorigin-panels.php

```diff
@@ -732,11 +732,14 @@ function siteorigin_panels_generate_css($post_id, $panels_data = false){
 			));
 		}
 
+
+		$collapse_order = !empty( $grid['style']['collapse_order'] ) ? $grid['style']['collapse_order'] : ( !is_rtl() ? 'left-top' : 'right-top' );
```

Comment:
I think once a user has manually set the collapse order, we should obey that fairly strictly. It's only when they've selected _Default_ that we'd want to use `is_rtl()` to work out the best option.